index.html

```diff
@@ -26,7 +26,7 @@ <h1>Add Tasks</h1>
         <input type="text" id="input" placeholder="Add a Todo" />
         <button id="addTaskButton">Add Task</button>
         <div class="trashInput">
-            <img src="icons/delete.svg" job="delete" id="0" />
+          <img src="icons/delete.svg" job="delete" id="0" />
```

Comment:
non standard attribute `job` used, consider using data attributes instead.

Reply:
removed as not required.